Doc/library/functions.rst

```diff
@@ -4,8 +4,8 @@
 Built-in Functions
 ==================
 
-The Python interpreter has a number of functions and types built into it that
-are always available.  They are listed here in alphabetical order.
```

Comment:
We generally use double spaces after periods between sentences.

Reply:
That's from formatting, didn't notice.